### PR TITLE
FAI-673: Counterfactual outcome selection does not disable button if the outcomes are restored to their original values

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.redhat.com/browse/FAI-673

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to replicate CI configuration locally?
</summary>

Build Chain tool does "simple" maven build(s), the builds are just Maven commands, but because the repositories relates and depends on each other and any change in API or class method could affect several of those repositories there is a need to use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
 
[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

</details>

### CODE REVIEW

**Review by kelvah — APPROVED** [review]
LGTM :+1:

**Comment by manstis** [comment]
Jenkins retest this

File: ui-packages/packages/trusty/src/components/Organisms/CounterfactualOutcomeSelection/tests/CounterfactualOutcomeSelection.test.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    originalValue: {
+      kind: 'UNIT',
+      type: 'number',
+      value: 100
+    }
+  }
+];
+
+const onClose = jest.fn();
+const verifyConfirmButtonState = (wrapper: ReactWrapper, state: boolean) => {
```
Comment (sjamboro) [inline ui-packages/packages/trusty/src/components/Organisms/CounterfactualOutcomeSelection/tests/CounterfactualOutcomeSelection.test.tsx]: @manstis can you rename this function? Based on usage of this function I suggest something like "verifyConfirmButtonIsDisabled". When I read tests I do not know which state represents "true" or "false". First time I thought that ```verifyConfirmButtonState(wrapper, true);``` means that the confirm button is enabled.

**Review by sjamboro — APPROVED** [review]

File: ui-packages/packages/trusty/src/components/Organisms/CounterfactualOutcomeSelection/tests/CounterfactualOutcomeSelection.test.tsx (hunk 1 above)
Comment (manstis) [reply ui-packages/packages/trusty/src/components/Organisms/CounterfactualOutcomeSelection/tests/CounterfactualOutcomeSelection.test.tsx]: Done. Thank-you for the recommendation.

**Pull request merged**